addons/sky_3d/src/Skydome.gd

```diff
@@ -5,30 +5,101 @@
 class_name Skydome
 extends Node
 
-
 signal sun_direction_changed(value)
 signal sun_transform_changed(value)
 signal moon_direction_changed(value)
 signal moon_transform_changed(value)
 signal day_night_changed(value)
 signal lights_changed
 
-
-enum SkyQuality {
-	Low, High
-}
-
-enum MoonResolution {
-	R64, R128, R256, R512, R1024,
-}
+@export_group("Global")
+@export_range(0.0, 1.0, 0.001) var tonemap_level: float = 0.0: set = set_tonemap_level
+@export var exposure: float = 1.3: set = set_exposure
+@export var ground_color: Color = Color(0.3, 0.3, 0.3, 1.0): set = set_ground_color
+@export var horizon_level: float = 0.0: set = set_horizon_level
+
+@export_group("Sun")
```

Comment:
Given the large nature of this file, let's move Sun exported variables down into the sun group of functions, and so on.